docs/docker/includes/intro.rst

```diff
@@ -2,7 +2,7 @@ eProsima provides the Fast DDS and the Fast DDS Suite Docker images for those wh
 platform.
 They can be downloaded from `eProsima's downloads page <https://eprosima.com/index.php/downloads-all>`_.
 
-This Docker images were built for Ubuntu 22.04 (Focal Fossa).
+This Docker images were built for Ubuntu 20.04 (Focal Fossa).
```

Comment:
I think that the most recent Fast DDS docker images are built for Ubuntu 22.04. Maybe the name is the one mistaken :wink:

Reply:
True, it is Jammy since 2.8.1